src/ngx_http_lua_connection.c

```diff
@@ -0,0 +1,353 @@
+/*
+ * Copyright (C) 2014-2015 Daurnimator
+ */
+
+/* need to include DDEBUG so that ngx_http_lua_util.h works */
+#ifndef DDEBUG
+#define DDEBUG 0
+#endif
+#include "ddebug.h"
+
+#include "ngx_http_lua_connection.h"
+
+#include <math.h> /* HUGE_VAL */
+#include <poll.h> /* POLLIN, POLLOUT */
+
+#include <lua.h>
+#include <lauxlib.h>
```

Comment:
I usually like being explicit here: If you use a declaration in a header; always include that directly (otherwise you get into a mess of implementation specific details).

I can remove it if it's against your style though :(